tools/pom.xml

```diff
@@ -2,154 +2,162 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
yep agree, a bit too much of formatting applied, reverted and only one additional dependency added.